docs/apps/by_discipline.md

```diff
@@ -75,6 +75,7 @@
 * [Amber](amber.md) Molecular dynamics suite
 * [AMS](ams.md) Modelling suite providing engines like ADF, BAND, DFTB and MOPAC
 * [AMS-GUI](ams-gui.md) AMS integrated GUI
+* [COSMO-RS](cosmors.md) Toolbox for predictive property calculation of liquids
```

Comment:
The page header is "COSMOsuite", which is automatically used in the automatically generated applications pages (by_license, index). For consistency, should we use the same here as well, i.e. instead of COSMO-RS?

```suggestion
* [COSMOsuite](cosmors.md) Toolbox for predictive property calculation of liquids
```

Reply:
Ok, if this is more consistent and conforms to the process. The product name COSMOsuite is quite unknown compared to COSMO-RS that is a common name for the model (cf. google search)

Reply:
Maybe then change the other way around, i.e. page header to COSMO-RS and the explicitly mention the package name in the intro?

Reply:
Sounds good!